### PR TITLE
Now we can create list of employees! Yeah.

### PR DESCRIPTION
**Pull request by asystentRoberta**

### CODE REVIEW

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/AddEmployeesRequest.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package pl.com.bohdziewicz.tutorials.mvcspring;
+
+import java.util.List;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter class AddEmployeesRequest {
```
Comment (qrczepieczone) [inline src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/AddEmployeesRequest.java]:
```suggestion
@Getter class EmployeesRequest{
```

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/Employee.java [hunk 2]
```diff
@@ -4,9 +4,11 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @AllArgsConstructor
+@NoArgsConstructor
```
Comment (qrczepieczone) [inline src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/Employee.java]: 👌 It's for json purpose.

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/EmployeesController.java [hunk 3]
```diff
@@ -45,4 +47,26 @@ Employee addEmployee(
 
         return newEmployee;
     }
+
```
Comment (qrczepieczone) [inline src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/EmployeesController.java]: There's a lot going on here.

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/AddEmployeesRequest.java (hunk 1 above)
Comment (asystentRoberta) [reply src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/AddEmployeesRequest.java]: Done.

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/Employee.java (hunk 2 above)
Comment (asystentRoberta) [reply src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/Employee.java]: Yeah.

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/EmployeesController.java (hunk 3 above)
Comment (asystentRoberta) [reply src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/EmployeesController.java]: I wanted to describe everything, but I have no time. 😞

**Pull request merged**